pkg/porter/runs_test.go

```diff
@@ -71,32 +71,36 @@ func TestPorter_PrintInstallationRunsOutput(t *testing.T) {
 	}
 
 	for _, tc := range outputTestcases {
-		p := NewTestPorter(t)
-		defer p.Close()
-		ctx := context.Background()
+		tc := tc
```

Comment:
this is a bit unreleated, but I saw that we were calling defer in a for loop and this just fixes that quick